kubernetes/active-passive/check-readiness.sh

```diff
@@ -10,9 +10,11 @@ podhealth() {
 
   activepod=$(for i in 0 1; do echo $podname-$i;done | grep -v $HOSTNAME)
 
-  echo "activepod = ${activepod}"
+  echo "activepod = ${activepod}.${serviceDomain}"
```

Comment:
where is service domain coming from ? is it K8 populated ?

Reply:
no this is a custom env variable.. do you know if there are defaults that k8s provides? had a look around and couldn't find a better way to get this